### PR TITLE
doc: Document how to increase operator's memory limits

### PR DESCRIPTION
**Pull request by jhrozek**
This has been suggested by the operator-sdk developers. See also:
    https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/subscription-config.md#resources
and:
    https://sdk.operatorframework.io/docs/best-practices/managing-resources/

### CODE REVIEW

**Comment by jhrozek** [comment]
@sheriff-rh it would be nice to incorporate this into our documentation somehow

**Comment by jhrozek** [comment]
@mkumku for awareness, although this is just a doc change

**Comment by jhrozek** [comment]
/label qa-approved
this is only a change for upstream docs, no need to waste QE resources

**Review by sheriff-rh — APPROVED** [review]
lgtm! If you create a separate jira sub task and assign it to William Harris, he's up to speed on this PR and ready to go.

**Comment by jhrozek** [comment]
> lgtm! If you create a separate jira sub task and assign it to William Harris, he's up to speed on this PR and ready to go.

Unfortunately the related Jira is OCPBUGSM-39585 which can't have subtasks (it's a clone from BZ). I can file a different card for William though, I pinged him on slack.

**Comment by jhrozek** [comment]
@Vincent056 @mrogers950 @rhmdnd PTAL

**Review by rhmdnd — APPROVED** [review]
/lgtm

File: doc/usage.md [hunk 1]
```diff
@@ -378,3 +378,29 @@ the operator will leave `PriorityClass` empty, issue a warning, and
 continue scheduling scans without a `PriorityClass`.
 
 [1]: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#priorityclass
+
+## Increasing operator's memory or CPU limits
+
+In some cases, the compliance-operator might require more memory than the default
+limits allow. If the operator had been installed via OLM (which is the case when
```
Comment (Wiharris) [inline doc/usage.md:385]: LGTM! Just one minor suggestion. 
```suggestion
limits allow. If the operator had been installed via OLM (which is the case when
```
limits allow. If the operator had been installed through OLM (which is the case when

**Comment by Wiharris** [comment]
LGTM!

**Comment by jhrozek** [comment]
The current version takes into account @Wiharris's suggestion. For some reason I didn't see the suggestion here in the GH UI, but I got an e-mail notification. Anyway, the suggestion was:
```
diff --git a/doc/usage.md b/doc/usage.md
index 37bd65a0..3162ed46 100644
--- a/doc/usage.md
+++ b/doc/usage.md
@@ -382,7 +382,7 @@ continue scheduling scans without a `PriorityClass`.
 ## Increasing operator's memory or CPU limits
 
 In some cases, the compliance-operator might require more memory than the default
-limits allow. If the operator had been installed via OLM (which is the case when
+limits allow. If the operator had been installed through OLM (which is the case when
 the operator is installed through the OCP Web Console), the best way is to set
 custom limits in the [Subscription object](https://github.com/operator-framework/operator-lifecycle-manager/blob/mas>
 

```

**Comment by jhrozek** [comment]
@rhmdnd can you re-add lgtm again, please?

**Pull request merged**